USAGE.md

```diff
@@ -5,16 +5,19 @@
     ```bash
     # Run in 3 terminals. If you want more logs, add `RUST_LOG=debug` before the command.
 
-    ./xline --name node1 --cluster-peers 127.0.0.1:2380 127.0.0.1:2381 --self-ip-port 127.0.0.1:2379 --leader-ip-port 127.0.0.1:2379 
+    ./xline --name node1 --members node1=127.0.0.1:2379,node2=127.0.0.1:2380,node3=127.0.0.1:2381
```

Comment:
Is here missing the `--is-leader` option?

Reply:
we can  run without this flag now, curp will elect a leader

Reply:
I guess we still need it. Because as we discussed yesterday, curp won't start an election automatically when it starts.

Reply:
Apart from the reasons that markcty has mentioned, I think it's a good idea to add this option, as doing so can provide developers who are green to Xline with more details on how to use it. Just my two cents.

Reply:
added